realm/src/main/java/io/realm/Realm.java

```diff
@@ -914,6 +914,7 @@ public void writeCopyTo(File destination) throws IOException {
      * <p>
      * @param destination File to save the Realm to
      * @throws java.io.IOException if any write operation fails
+     * @throws RealmEncryptionNotSupportedException if the device doesn't support Realm encryption.
```

Comment:
@bmunkholm I changed that :) full path is removed as you suggested (importing on the top instead)